transformations/replace_financial_amounts/README.md

```diff
@@ -0,0 +1,80 @@
+# Replace Financial amounts 🦎  + ⌨️ → 🐍
+This transformation replaces consistently financial amounts throughout a text.
+The replacement changes the amount, the writing format as well as the currency of the financial amount.
+The change is consistent in regard to:
+- the modifier used to change all amounts of the same currency throughout the text.
```

Comment:
The change is consistent with respect to:

Reply:
changed